.github/workflows/tests.yml

```diff
@@ -136,7 +146,7 @@ jobs:
       # set individual cache key
       cache_key: ${{ matrix.os }}-py${{ matrix.python-version }}-${{ contains(matrix.options, 'mindep') }}-${{ contains(matrix.options, 'mindepversion') }}
       # set conda environment file with dependencies
-      env_file: .github/test${{ contains(matrix.options, 'mindepversion') && '_mindepversion' || contains(matrix.options, 'mindep') && '_mindep' || '' }}_conda_env.yml
+      env_file: .github/test${{ contains(matrix.options, 'np2') && '_np2' || contains(matrix.options, 'mindepversion') && '_mindepversion' || contains(matrix.options, 'mindep') && '_mindep' || '' }}_conda_env.yml
```

Comment:
will this remain when the merge is done ? or should the mindep still be tested in the future?

Reply:
> mindep still be tested in the future?

mindep is still being tested, matrix was just made bigger with additional runners for np2. this line only sets up the filename for the conda env file used in the respective runner, depending on the "flags"

Reply:
After numpy releases v2.0.0 we can probably kick out these additional runs again, since our "normal" current dependency version runners will eventually pick up numpy v2. It's just for the time while there is only the v2.0.0rc2 prerelease to test with we need some extra specs in the env file so that the pre release gets picked up